### ISSUE TITLE
I don't want to use tls. What should I do?

### ISSUE THREAD

**Issue by iicemeta** [6Kmfi6HP/EDtunnel#51]
Copilot can't be opened when the new code uses tls, but the old code can. I don't want to use tls. How can I close it?

![Screenshot_2024-12-04-07-54-14-513_com.microsoft.copilot.jpg](https://github.com/user-attachments/assets/c0201230-7082-4829-b127-ced37b558d24)

![Screenshot_2024-12-04-07-54-32-258_com.v2ray.ang.jpg](https://github.com/user-attachments/assets/11500d72-323f-4277-8c67-da2e607a98b4)

**Comment by 6Kmfi6HP** [comment]
Use workers instead of pages

**Comment by iicemeta** [comment]
Thank you very much, I successfully deployed the notls node, but unfortunately, the problem still exists.
I thought it was tls that prevented me from opening other applications.But after using notls, the problem still exists, and I still can't open applications such as GitHub, Google play and copilot.
I tried to change DNS, but it didn't seem to work
This problem only appears on my mobile phone, and my computer can still open them.
![Screenshot_2024-12-04-21-54-38-519_com.v2ray.ang.jpg](https://github.com/user-attachments/assets/0bc52ead-64e4-4231-91e7-9f84665f634d)

![Screenshot_2024-12-04-21-54-16-761_com.microsoft.copilot.jpg](https://github.com/user-attachments/assets/68c6e6b2-3513-4443-b335-16bf01d73823)

![Screenshot_2024-12-04-21-53-55-452_com.android.vending.jpg](https://github.com/user-attachments/assets/515ad149-cd9b-4bde-b89a-718e8152bf5a)

![Screenshot_2024-12-04-21-59-45-825_com.v2ray.ang.jpg](https://github.com/user-attachments/assets/48ee1ccc-2185-4022-84bd-f1ee40517626)

**Comment by RichardMiku** [comment]
Thank you very much, I successfully deployed the notls node, but unfortunately, the problem still exists.
I thought it was tls that prevented me from opening other applications.But after using notls, the problem still exists, and I still can't open applications such as GitHub, Google play and copilot.
I tried to change DNS, but it didn't seem to work
This problem only appears on my mobile phone, and my computer can still open them.
![Screenshot_2024-12-04-21-54-38-519_com.v2ray.ang.jpg](https://github.com/user-attachments/assets/0bc52ead-64e4-4231-91e7-9f84665f634d)

![Screenshot_2024-12-04-21-54-16-761_com.microsoft.copilot.jpg](https://github.com/user-attachments/assets/68c6e6b2-3513-4443-b335-16bf01d73823)

![Screenshot_2024-12-04-21-53-55-452_com.android.vending.jpg](https://github.com/user-attachments/assets/515ad149-cd9b-4bde-b89a-718e8152bf5a)

![Screenshot_2024-12-04-21-59-45-825_com.v2ray.ang.jpg](https://github.com/user-attachments/assets/48ee1ccc-2185-4022-84bd-f1ee40517626)
